### DISCUSSION TITLE
Add a roadmap for the project

### DISCUSSION THREAD

**Discussion by yuzhouu** [facebook/lexical#2019] [General]
Need a roadmap for the project, so the community can track when and what will happen next.

**Comment by Proful** [comment]
Are you looking for this?
https://github.com/facebook/lexical/projects/1

**Reply by yuzhouu** [reply]
It's only the things on board, I wish the team can share their vision, like when will 1.0.0 be published, what will be included in 1.0.0, and etc.

**Comment by Nantris** [comment]
Is there anywhere else we can consult now that that board is archived?

**Comment by Nantris** [comment]
Friendly bump. 

It's been about 2 months of pondering and I feel pretty settled that we want to move forward with Lexical but that's really tough given that the docs are incomplete and there's no roadmap.

There are a few things we don't need on day one but we do need to be able to actually port our existing editor to Lexical _eventually_ and we need to know that those things are going to happen _eventually._

A list of those things would be:

1. More complete docs so we can actually be certain of what is/is not possible
2. Ability to add decoration nodes without affecting serialized output
  a. The developer experience of manually removing and adding these nodes would be a pain, to say nothing of the nightmare that would present for performance (especially mobile)
3. A way to render non-editable content within the editor - similar to how the codeblock uses a portal to show the `CodeActionMenuPlugin` - but ideally it could be done without portal. This may be possible already but because the docs are incomplete and it doesn't seem to occur in Playground, I'm left to wonder

I think those are really our only sticking points. If \#1 could be addressed soon and \#2/\#3 are planned then we could feel much more safe making this leap from Slate which we currently use.

Can anyone comment? @trueadm @zurfyx @acywatson?
